src/service/user.js

```diff
@@ -0,0 +1,48 @@
+class User {
+  constructor() {
+    this.user = new Set();
+  }
+
+  create = (data) => {
+    const { user } = this;
+    const object = {
+      id: user.size + 1,
+      ...data,
+    };
+    user.add(object);
+    return object;
+  };
+
+  update = (data) => {
+    const { id, update } = data;
+    const { user } = this;
+    const updatedUser = Object.assign(
+      [...user].find((user) => user.id === Number(id)),
```

Comment:
[...user].find((user) => user.id === Number(id)) why you have to use rest property for user?
user.find((user) => user.id === Number(id))